src/lib/dialog/dialog.ts

```diff
@@ -125,6 +127,20 @@ export class MdDialog {
 
     return state;
   }
+
+  /**
+   * Applies default options to the dialog config.
+   * @param dialogConfig Config to be modified.
+   * @returns The new configuration object.
+   */
+  private _applyConfigDefaults(dialogConfig: MdDialogConfig): MdDialogConfig {
+    let defaults: MdDialogConfig = {
+      role: 'dialog',
+      disableClose: false
```

Comment:
Why not leave the defaults in `MdDialogConfig` and simply say `new MdDialogConfig` here?